{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2014-05-14T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2017-01-04T17:57:01",
                "orgId": "cc1ad9ee-3454-478d-9317-d3e869d708bc",
                "shortName": "canonical"
            },
            "references": [
                {
                    "name": "61281",
                    "tags": [
                        "third-party-advisory",
                        "x_refsource_SECUNIA"
                    ],
                    "url": "http://secunia.com/advisories/61281"
                },
                {
                    "name": "DSA-2934",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_DEBIAN"
                    ],
                    "url": "http://www.debian.org/security/2014/dsa-2934"
                },
                {
                    "name": "USN-2212-1",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_UBUNTU"
                    ],
                    "url": "http://ubuntu.com/usn/usn-2212-1"
                },
                {
                    "name": "openSUSE-SU-2014:1132",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_SUSE"
                    ],
                    "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html"
                },
                {
                    "name": "[oss-security] 20140514 Re: CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://www.openwall.com/lists/oss-security/2014/05/15/3"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued/"
                },
                {
                    "name": "[oss-security] 20140514 CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://www.openwall.com/lists/oss-security/2014/05/14/10"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "security@ubuntu.com",
                    "ID": "CVE-2014-1418",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly include the (1) Vary: Cookie or (2) Cache-Control header in responses, which allows remote attackers to obtain sensitive information or poison the cache via a request from certain browsers."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "61281",
                            "refsource": "SECUNIA",
                            "url": "http://secunia.com/advisories/61281"
                        },
                        {
                            "name": "DSA-2934",
                            "refsource": "DEBIAN",
                            "url": "http://www.debian.org/security/2014/dsa-2934"
                        },
                        {
                            "name": "USN-2212-1",
                            "refsource": "UBUNTU",
                            "url": "http://ubuntu.com/usn/usn-2212-1"
                        },
                        {
                            "name": "openSUSE-SU-2014:1132",
                            "refsource": "SUSE",
                            "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html"
                        },
                        {
                            "name": "[oss-security] 20140514 Re: CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                            "refsource": "MLIST",
                            "url": "http://www.openwall.com/lists/oss-security/2014/05/15/3"
                        },
                        {
                            "name": "https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued/",
                            "refsource": "CONFIRM",
                            "url": "https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued/"
                        },
                        {
                            "name": "[oss-security] 20140514 CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                            "refsource": "MLIST",
                            "url": "http://www.openwall.com/lists/oss-security/2014/05/14/10"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-06-30T07:10:48.450Z"
                },
                "references": [
                    {
                        "name": "61281",
                        "tags": [
                            "third-party-advisory",
                            "x_refsource_SECUNIA",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://secunia.com/advisories/61281/DEMO_2024_06_19"
                    },
                    {
                        "name": "DSA-2934",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_DEBIAN",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://www.debian.org/security/2014/dsa-2934/DEMO_2024_06_19"
                    },
                    {
                        "name": "USN-2212-1",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_UBUNTU",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://ubuntu.com/usn/usn-2212-1/DEMO_2024_06_19"
                    },
                    {
                        "name": "openSUSE-SU-2014:1132",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_SUSE",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html/DEMO_2024_06_19"
                    },
                    {
                        "name": "[oss-security] 20140514 Re: CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://www.openwall.com/lists/oss-security/2014/05/15/3/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued//DEMO_2024_06_19"
                    },
                    {
                        "name": "[oss-security] 20140514 CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "http://www.openwall.com/lists/oss-security/2014/05/14/10/DEMO_2024_06_19"
                    },
                    {
                        "name": "61281",
                        "tags": [
                            "third-party-advisory",
                            "x_refsource_SECUNIA",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://secunia.com/advisories/61281/adp-test-6-28"
                    },
                    {
                        "name": "DSA-2934",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_DEBIAN",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://www.debian.org/security/2014/dsa-2934/adp-test-6-28"
                    },
                    {
                        "name": "USN-2212-1",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_UBUNTU",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://ubuntu.com/usn/usn-2212-1/adp-test-6-28"
                    },
                    {
                        "name": "openSUSE-SU-2014:1132",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_SUSE",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://lists.opensuse.org/opensuse-updates/2014-09/msg00023.html/adp-test-6-28"
                    },
                    {
                        "name": "[oss-security] 20140514 Re: CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://www.openwall.com/lists/oss-security/2014/05/15/3/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://www.djangoproject.com/weblog/2014/may/14/security-releases-issued//adp-test-6-28"
                    },
                    {
                        "name": "[oss-security] 20140514 CVE Reuest: Django: Malformed URLs from user input incorrectly validated",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "http://www.openwall.com/lists/oss-security/2014/05/14/10/adp-test-6-28"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "cc1ad9ee-3454-478d-9317-d3e869d708bc",
        "assignerShortName": "canonical",
        "cveId": "CVE-2014-1418",
        "datePublished": "2014-05-16T15:00:00",
        "dateReserved": "2014-01-13T00:00:00",
        "dateUpdated": "2024-06-30T07:10:48.450Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}